src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -230,10 +225,37 @@ public static JsonRpc Attach(Stream sendingStream, Stream receivingStream, objec
         }
 
         /// <summary>
+        /// Adds the specified targets as possible objects to invoke when incoming messages are received.
+        /// </summary>
+        /// <param name="targets">Targets to invoke when incoming messages are received.</param>
+        /// <remarks>This method must be called before JsonRpc starts listening for messages.</remarks>
+        public void AddLocalRpcTarget(params object[] targets)
```

Comment:
Your method takes a plural argument yet your method name suggests singular.
We can rename the method, or take a singular parameter. I'm leaning toward a singular parameter, myself, based on my belief that most callers will have just one target to "add" so we'll avoid a loop and an array allocation by just taking one. And callers would then be responsible to avoid calling it with a null argument.